packages/components/src/internal/components/samples/SamplesTabbedGridPanel.tsx

```diff
@@ -256,6 +260,29 @@ export const SamplesTabbedGridPanel: FC<Props> = memo(props => {
         [createNotification, dismissNotifications, getSampleAuditBehaviorType]
     );
 
+    const onPrintLabel = useCallback((modelId: string): void => {
+        const _model = queryModels[modelId] ?? activeModel;
+        setPrintDialogModel(_model);
+        setShowPrintModal(true);
+    },[queryModels, activeModel]);
+
+    const onLabelExport = { [EXPORT_TYPES.LABEL]: onPrintLabel };
+
+    const onCancelPrint = useCallback(():void => {
+        setPrintDialogModel(undefined);
+        setShowPrintModal(false);
+    }, []);
+
+    const afterPrint = useCallback((numSamples: number, numLabels: number): void => {
+        setPrintDialogModel(undefined);
```

Comment:
looks like `setShowPrintModal` is never set back to `false` here. I think that state variable can actually just go away and just use the `printDialogModel` state var instead. So it would show the modal if the `printDialogModel` is defined and then setting it back to undefined would hide the modal.